src/load/azext_load/data_plane/utils/argtypes.py

```diff
@@ -34,6 +34,14 @@
     help="Do not wait for the long-running operation to finish.",
 )
 
+disable_public_ip = CLIArgumentType(
+    validator=validators.validate_disable_public_ip,
+    options_list=["--disable-public-ip"],
+    type=str,
+    help="Disable the deployment of a public IP address, load balancer, and network security group while testing a private endpoint.",
+)
```

Comment:
do we need to mention deployment in the customer's subnet

Reply:
keeping it consistent with portal